src/util.ts

```diff
@@ -750,6 +765,137 @@ export async function prepareXchainFromAmountCalculation({
 	}
 }
 
+async function routeWithSlippage({
```

Comment:
nit: name of this function doesn't  seem super  descriptive? Maybe should have a verb? `estimateRouteWithMinSlippage` ?

Reply:
done